sdks/go/test/integration/integration.go

```diff
@@ -214,7 +216,9 @@ func CheckFilters(t *testing.T) {
 		}
 	}
 	// TODO(lostluck): Improve default job names.
```

Comment:
```suggestion
```

Reply:
I didn't originally get rid of this because I wasn't sure if this fully completed it, but at the same time it is probably not helpful if its unclear so I'm good to remove it